src/test/resources/db/schema/schema.sql

```diff
@@ -1,56 +1,58 @@
- create table history (
-        created_at datetime(6),
-        id bigint not null auto_increment,
-        batch_id varchar(36),
-        delivery_id varchar(36),
-        message_id varchar(36) not null,
-        party_id varchar(36),
-        content LONGTEXT,
-        department varchar(255),
-        issuer varchar(255),
-        municipality_id varchar(255),
-        origin varchar(255),
-        status_detail LONGTEXT,
-        message_type enum ('DIGITAL_INVOICE','DIGITAL_MAIL','EMAIL','LETTER','MESSAGE','SLACK','SMS','SNAIL_MAIL','WEB_MESSAGE'),
-        original_message_type enum ('DIGITAL_INVOICE','DIGITAL_MAIL','EMAIL','LETTER','MESSAGE','SLACK','SMS','SNAIL_MAIL','WEB_MESSAGE'),
-        status enum ('AWAITING_FEEDBACK','FAILED','NOT_SENT','NO_CONTACT_SETTINGS_FOUND','NO_CONTACT_WANTED','PENDING','SENT'),
-        primary key (id)
-    ) engine=InnoDB;
-
-    create table messages (
-        created_at datetime(6),
-        id bigint not null auto_increment,
-        batch_id varchar(255),
-        content LONGTEXT NOT NULL,
-        delivery_id varchar(255),
-        issuer varchar(255),
-        message_id varchar(255),
-        municipality_id varchar(255),
-        origin varchar(255),
-        party_id varchar(255),
-        message_type enum ('DIGITAL_INVOICE','DIGITAL_MAIL','EMAIL','LETTER','MESSAGE','SLACK','SMS','SNAIL_MAIL','WEB_MESSAGE'),
-        original_message_type enum ('DIGITAL_INVOICE','DIGITAL_MAIL','EMAIL','LETTER','MESSAGE','SLACK','SMS','SNAIL_MAIL','WEB_MESSAGE'),
-        status enum ('AWAITING_FEEDBACK','FAILED','NOT_SENT','NO_CONTACT_SETTINGS_FOUND','NO_CONTACT_WANTED','PENDING','SENT'),
-        primary key (id)
-    ) engine=InnoDB;
-
-    create index idx_history_batch_id
-       on history (batch_id);
-
-    create index idx_history_message_id
-       on history (message_id);
-
-    create index idx_history_delivery_id
-       on history (delivery_id);
-
-    create index idx_history_origin
-       on history (origin);
-
-    create index idx_history_issuer
-       on history (issuer);
-
-    create index idx_history_department
-       on history (department);
-
-    create index idx_history_municipality_id
-       on history (municipality_id);
+create table history (
+     created_at datetime(6),
+     id bigint not null auto_increment,
+     batch_id varchar(36),
+     delivery_id varchar(36),
+     message_id varchar(36) not null,
+     party_id varchar(36),
+     content LONGTEXT,
+     department varchar(255),
+     destination_address varchar(255),
```

Comment:
Should be `longtext`, or just `text` if you decide to change?

Reply:
Maybe, or maybe not, since the schema.sql file is "generated" from the actual JPA annotations on entities, and there's no correspondence to the actual db schema being managed through Flyway scripts. Imho, either the schema verification test should be removed or we should update all entity annotations to reflect the Flyway-migrated schema